examples/Manifest.toml

```diff
@@ -172,8 +172,8 @@ uuid = "c8b6d40d-e815-466f-95ae-c48aefa668fa"
 version = "0.7.0"
 
 [[deps.ClimaTimeSteppers]]
-deps = ["CUDA", "DiffEqBase", "KernelAbstractions", "LinearAlgebra", "MPI", "SciMLBase", "StaticArrays"]
-git-tree-sha1 = "4e49bf1b897d233c914acbcf381505f4788e0b6b"
+deps = ["CUDA", "DiffEqBase", "DiffEqCallbacks", "KernelAbstractions", "LinearAlgebra", "MPI", "SciMLBase", "StaticArrays"]
+path = "/home/valeria/.julia/dev/ClimaTimeSteppers"
```

Comment:
woops! I meant to use the `dev` version of the package but not to have my dev path